Gems/LyShine/Code/Source/LyShineLoadScreen.cpp

```diff
@@ -97,7 +98,9 @@ namespace LyShine
             return false;
         }
 
-        if (!gEnv || !gEnv->pRenderer || !gEnv->pLyShine)
+        //TODO: gEnv->pRenderer is always null, fix the logic below
+        AZ_ErrorOnce(nullptr, gEnv && gEnv->pLyShine, "NotifyLevelLoadStart needs to be removed/ported to use Atom");
+        if (!gEnv || true || !gEnv->pLyShine)
```

Comment:
As above, maybe better to just remove the whole if statement and return false?

Reply:
Both of these cases are ones I'm not 100% should be trimmed so harshly, unless the loading screen functionality will change?

Reply:
Think it's fine to leave all the code underneath if it might be relevant in future, just think it's a little confusing having an if statement that always returns true. Maybe:

AZ_ErrorOnce(...
return false;

if (!gEnv || !gEnv->pLyShine)
...

?

(gEnv will hopefully also go away at some point).

Reply:
I've replaced it with suggested `return false` and moved original code into `#if 0` blocks ( 7f9557b )

Reply:
Thanks!